stix_shifter/stix_transmission/src/modules/utils/RestApiClient.py

```diff
@@ -0,0 +1,89 @@
+import requests
+import sys
+import base64
+import ssl
+import json
+import collections
+import urllib.parse
+
+# This is a simple HTTP client that can be used to access the REST API
+class RestApiClient:
+
+    def __init__(self, host, port = None, cert=None, headers={}, urlModifierFunction=None):
+        server_ip = host
+        if port is not None:
+            server_ip += ":" + str(port)            
+        self.server_ip = server_ip
+        self.cert_file = None
+        if cert is not None:
+            # put cert/key pair into a file to read it later
+            cert_file_name = "cert.pem"
+            certfile = open(cert_file_name, "w+")
+            certfile.write(cert)
+            certfile.close()
+
+            with open(cert_file_name, 'w+') as f:
+                try:
+                    f.write(cert)
+                    f.close()
+                except IOError:
+                    print('Failed to setup certificate')
+
+            self.cert_file = cert_file_name
```

Comment:
We want to delete "cert.pem" somewhere after this... `os.remove(cert_file_name)`